mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -322,6 +322,7 @@ func resourceMongoDBAtlasCluster() *schema.Resource {
 					},
 				},
 			},
+			"snapshot_backup_policy": computedCloudProviderSnapshotBackupPolicySchema(),
```

Comment:
FYI I _think_ we can compute this even if they user is not using provider_backup_enabled = true but we should probably test it to be sure.  Basically the snapshot backup policy is only applicable if provider_backup_enabled = true.  backup_enabled does not use this policy.

Reply:
yes it must even be computed if the user is no using the provider_backup_enabled = true

Reply:
Will this not result in an error?    Based on what I tested the API for cloud provider snapshots returns an error "BACKUP_CONFIG_NOT_FOUND" unless provider_backup_enabled is true.

Reply:
Don't worry that's covered, I validated that type of error :), if it happens just the snapshot_backup_policy won't be fill, it keeps empty.

Reply:
Ah, then we should be fine for now then as eventually we'll have all clusters using cps.